http-core/src/main/scala/org/apache/pekko/http/scaladsl/settings/Http2ServerSettings.scala

```diff
@@ -136,6 +153,7 @@ object Http2ServerSettings extends SettingsCompanion[Http2ServerSettings] {
     require(minCollectStrictEntitySize <= (incomingConnectionLevelBufferSize / maxConcurrentStreams),
       "min-collect-strict-entity-size <= incoming-connection-level-buffer-size / max-concurrent-streams")
     require(outgoingControlFrameBufferSize > 0, "outgoing-control-frame-buffer-size must be > 0")
+    require(frameTypeThrottleInterval.toMillis > 0, "frame-type-throttle.interval must be a positive duration")
```

Comment:
I still think this is better to not allow bad values - the default value is ok and we have a warning in the reference.conf.

And it is more efficient to do 1 check instead of 2 checks.

Reply:
OK's let's ship it.